docs/configuration.md

```diff
@@ -25,13 +25,17 @@ from flask_pyoidc.provider_configuration import ProviderConfiguration, ProviderM
 
 provider_metadata = ProviderMetadata(issuer='https://op.example.com',
                                      authorization_endpoint='https://op.example.com/auth',
-                                     jwks_uri='https://op.example.com/jwks')
+                                     jwks_uri='https://op.example.com/jwks',
+                                     userinfo_endpoint='https://op.example.com/userinfo')
 config = ProviderConfiguration(provider_metadata=provider_metadata, [client configuration])
 ```
 
 See the OpenID Connect specification for more information about the
 [provider metadata](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata).
 
+As mentionned in OpenID Connect specification, `userinfo_endpoint` is optional. If it's
```

Comment:
Small typo: "mentionned" -> "mentioned"

Reply:
ahah, yes, I fixed  it, french habit to put a second consonant before a vowel